guidance/03-analysis/analysis-steps/02-analysis-modelling.qmd

```diff
@@ -7,8 +7,64 @@ image-alt: Photo by Shubham Dhage on Unsplash
 
 In this step, we turn our data into insights. The work done here can vary. It can include statistical analysis, both modelled and descriptive, as well as more sophisticated Machine Learning (AI) – the choice being dependent on the output requirements of the customer. The output can be presented in the form of a report, a dashboard, or a slide deck.
 
-## Conducting the work
-Starting from the source tables, we want to create the agreed outputs. Depending on the work, there are many approaches and tools we can use to do this.
+## Selection of methods
+
+The analysis first requires an appropriate method that can provide the desired outputs.
+
+*	Create a short-list of methods that could be applicable to the problem.
+*	Consider the pros and cons of each candidate method:
+  *	are there examples of this method being applied to the problem archetype?
+  *	will the available data support the method? If not, is it feasible to transform the data into a more suitable form?
+  *	will the tools available support the method?
+*	Decide which methods are best suited to the problem:
+  *	often the best method will be clear
+  *	sometimes you may need to try a small number of methods before deciding on the overall best one
+
+Keep notes of your decision-making and method trials for the final write-up of the initiative.
+
+## Application of methods
+
+Apply the selected model(s) with **RIGOUR**; it should be **R**epeatable, **I**ndependent, **G**rounded in reality, **O**bjective, **U**ncertainty-managed, and **R**obust.
+
+* **Repeatable**: For an analytical process to be considered ‘valid’ it might reasonably be expected that for the “same” inputs and constraints the analysis produces the “same” outputs. It is important to note that different analysts will consider the analytical problem differently, potentially resulting in differing results, however if any one approach is repeated the results should be as expected.
```

Comment:
different type of quotations here. Should that be or should they be consistent?

Reply:
Removed quotes from "same" x2, seemed unnecessary.